definitions/checks/foreman_tasks/not_running.rb

```diff
@@ -9,8 +9,17 @@ class NotRunning < ForemanMaintain::Check
     end
 
     def run
-      assert(feature(:foreman_tasks).running_tasks_count == 0,
-             'There are actively running tasks in the system')
+      task_count = feature(:foreman_tasks).running_tasks_count
+      assert(task_count == 0,
+             failure_message(task_count),
+             :next_steps =>
+               [Procedures::ForemanTasks::FetchTasksStatus.new(:state => 'running'),
+                Procedures::ForemanTasks::UiInvestigate.new('search_query' => 'state = running')])
```

Comment:
The search query should be `state = running AND label !^(Features::ForemanTasks::EXCLUDE_ACTIONS_FOR_RUNNING_TASKS.join(" "))`

Reply:
The `UiInvestigate` procedure is using `press ENTER after the paused tasks are resolved` wording, which is not precise for this case. Using `press ENTER after the tasks are resolved` would be matching both running and paused tasks.

Reply:
Good Catch!! :+1: